### PR TITLE
Display XP progress on items

### PR DESCRIPTION
**Pull request by kyleshay**
Now that we're good at calculating the XP for items, we should consider displaying XP progress on the items themselves, as a progress bar or something. Open to ideas on how to represent this in a non-obtrusive way, especially when "primary stat" is being shown.

### CODE REVIEW

**Comment by bhollis** [comment]
I actually like it on the top! That's definitely what I was thinking of
doing.

**Comment by joshmedeski** [comment]
Between all the combinations of settings, I think it will look best on the top to balance all that's happening on the bottom of each box.

**Comment by kyleshay** [comment]
moved to the top border:

screen shot 2016-05-10 at 12 08 44 pm

**Comment by bhollis** [comment]
I love it!

File: Gruntfile.js [hunk 1]
```diff
@@ -2,6 +2,9 @@ module.exports = function(grunt) {
   var pkg = grunt.file.readJSON('package.json');
   var betaVersion = pkg.version.toString() + "." + (Math.floor(Date.now() / 60000) - 24365210);
 
+  grunt.loadNpmTasks('grunt-contrib-uglify');
+  grunt.loadNpmTasks('grunt-contrib-concat');
```
Comment (bhollis) [inline Gruntfile.js]: Would you mind putting the grunt changes into a separate PR? Then I can merge just the XP bar stuff.

File: app/scss/_style.scss [hunk 2]
```diff
@@ -811,6 +811,14 @@ img {
     position: absolute;
     z-index: 2;
   }
+
+  .item-xp-bar-small {
+    position: absolute;
+    top: 0px;
+    height: 2px;
+    opacity: .95;
```
Comment (bhollis) [inline app/scss/_style.scss]: Why?

File: Gruntfile.js (hunk 1 above)
Comment (kyleshay) [inline Gruntfile.js]: Shoot. Didn't mean to merge that. I'll clean it up in the am.

File: app/scss/_style.scss (hunk 2 above)
Comment (bhollis) [inline app/scss/_style.scss]: More specifically, why the opacity? It's always going to be against a white background.

**Comment by kyleshay** [comment]
removed unnecessary stuff, moved bar into div that controls opacity during search.

**Comment by bhollis** [comment]
I'm still seeing the Grunt stuff in this PR

**Comment by bhollis** [comment]
Should this be a setting?

Would you mind adding this to the CHANGELOG, either way?

**Comment by kyleshay** [comment]
done-- could go either way for the setting

**Comment by bhollis** [comment]
Merged. I'll use it for a while and see if it needs to be a setting.

**Comment by JosephBono** [comment]
Saw this in the BETA build yesterday, found it useful for differentiating which weapons I had put XP into to transfer over when turning in bounties.

**Pull request merged**